js-exercises/accessor-properties/accessorProperties.js

```diff
@@ -0,0 +1,28 @@
+function accessorProperties() {
+  const newObject = {};
+  Object.defineProperties(newObject, {
+    _number: {
+      value: null,
+      writable: true
+    },
+    number: {
+      get() {
+        if (typeof this._number === "number") {
+          return parseInt(this._number.toString(2), 10);
+        }
```

Comment:
if the prop was created like whats in the above comment, here instead of 'this._number' we could directly use 'this.value'. If you want you can refer to the following PR:
[https://github.com/balajikamalesh/pesto-exercise-starter/pull/11/commits/a3dbd3a7c6a23fc5ceafebe3c7fd6d8c25fc4280](url)